electroncash/daemon.py

```diff
@@ -138,7 +138,7 @@ def get_rpc_credentials(config):
 
 class Daemon(DaemonThread):
 
-    def __init__(self, config, fd, is_gui, plugins):
+    def __init__(self, config, fd, is_gui, plugins, listen_jsonrpc=True):
```

Comment:
```suggestion
    def __init__(self, config, fd, is_gui, plugins, *, listen_jsonrpc=True):
```

I suggest making this a kw-only arg since too many positional args make me nervous. :)  What do you think?

Reply:
Yes, I was thinking of the same! Ideally we could clean this up, because is_gui is unused, plugins can be refactored, etc. But good for now